lxd/db/images_test.go

```diff
@@ -59,3 +59,37 @@ func TestImageExists(t *testing.T) {
 
 	assert.True(t, exists)
 }
+
+func TestGetImage(t *testing.T) {
+	cluster, cleanup := db.NewTestCluster(t)
+	defer cleanup()
+
+	// public image with 'default' project
+	err := cluster.CreateImage(
+		"default", "abcd1", "x.gz", 16, true, false, "amd64", time.Now(), time.Now(), map[string]string{}, "container")
```

Comment:
@masnax tiny nit, not a blocker at all, but @stgraber has mentioned in the past that we should not overflow function calls onto multiple lines to aid in "grepping".

Reply:
Ah I see, I just copied the line down from another test. I'll fix that :)

Reply:
Yeah its certainly not applied everywhere (yet).